src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -13,8 +13,10 @@
 #    include <pwd.h>
 #    if defined(__FreeBSD__)
 #        include <stddef.h>
+// clang format off
```

Comment:
I'd like to see a comment that explains why this is necessary here, so others who stumble upon it understand.